client/ayon_core/hosts/houdini/api/lib.py

```diff
@@ -827,41 +839,47 @@ def get_camera_from_container(container):
     return cameras[0]
 
 
-def get_current_context_template_data_with_asset_data():
-    """
-    TODOs:
-        Support both 'assetData' and 'folderData' in future.
+def get_current_context_template_data_with_folder_attrs():
     """
 
+    Output contains 'folderAttributes' key with folder attribute values.
+
+    Returns:
+         dict[str, Any]: Template data to fill templates.
+
+    """
     context = get_current_context()
     project_name = context["project_name"]
-    asset_name = context["folder_path"]
+    folder_path = context["folder_path"]
     task_name = context["task_name"]
     host_name = get_current_host_name()
 
-    anatomy = Anatomy(project_name)
-    project_doc = get_project(project_name)
-    asset_doc = get_asset_by_name(project_name, asset_name)
+    project_entity = ayon_api.get_project(project_name)
+    anatomy = Anatomy(project_name, project_entity=project_entity)
+    folder_entity = ayon_api.get_folder_by_path(project_name, folder_path)
+    task_entity = ayon_api.get_task_by_name(
+        project_name, folder_path["id"], task_name
```

Comment:
There's a typo here.
```suggestion
        project_name, folder_entity["id"], task_name
```